must-gather/README.md

```diff
@@ -6,7 +6,7 @@ that expands its capabilities to gather Openshift Container Storage 4 informatio
 
 ### Usage
 ```sh
-oc adm must-gather --image=ashishranjan738/ocs-must-gather
+oc adm must-gather --image=quay.io/ocs-dev/ocs-must-gather
```

Comment:
This, too should go into a image-changing PR, imho.

Reply:
A PR by @kshlm is raised for this so marking this as resolved.